app/routes/project/posts/post.js

```diff
@@ -32,6 +32,16 @@ export default Ember.Route.extend({
   },
 
   actions: {
+    save(post) {
+      post.save().catch((error) => {
```

Comment:
@WenInCode I actually really don't understand what's going on in the test. As it currently stands, everything passes. If you think it's OK, I trust you.